src/NBitcoin/Network.cs

```diff
@@ -248,7 +256,8 @@ public NetworkOptions NetworkOptions
 
         private Network()
         {
-            this.genesis = new Block();
+            this.seeds = new List<DNSSeedData>();
```

Comment:
Adding this overrides what we have on 243 and 244 so it is probably a good idea to remove

```
= new List<DNSSeedData>();
 = new List<NetworkAddress>();
```

from there

Reply:
Ah yeah ofcourse thanks.